code/__DEFINES/dcs/signals.dm

```diff
@@ -262,9 +262,9 @@
 
 /////////////////
 
-///from base of area/Entered(): (/area)
+///from base of area/Entered(): (/area). Sent to all contents that "area-sensitive", see __DEFINES/traits.dm for info.
 #define COMSIG_ENTER_AREA "enter_area"
-///from base of area/Exited(): (/area)
+///from base of area/Exited(): (/area). Idem.
```

Comment:
What?

Reply:
Okay looking it up it's a research term that looks like it means "ditto"...can you just copy and paste what you wrote above? 😅

Reply:
Whoops.